src/app/forum/containers/thread/thread.component.ts

```diff
@@ -172,32 +184,85 @@ export class ThreadComponent implements AfterViewInit, OnDestroy {
     );
 
     this.subscriptions.add(
-      this.isThreadStatusOpened$.pipe(delay(0)).subscribe(isThreadStatusOpened => {
-        if (!isThreadStatusOpened) {
-          this.form.get('messageToSend')?.disable();
+      combineLatest([
+        this.threadStatus$.pipe(delay(0)),
+        this.disableControls$,
+      ]).subscribe(([ status, disableControls ]) => {
+        if (status?.data && (status.data.open || status.data.canOpen) && !disableControls) {
+          this.form.get('messageToSend')?.enable();
+          this.focusOnInput();
           return;
         }
-        this.form.get('messageToSend')?.enable();
-        this.focusOnInput();
+        this.form.get('messageToSend')?.disable();
```

Comment:
same here, a `else` would be more readable

Reply:
Fixed